### ISSUE TITLE
"Since my last review" doesn't work for the author

### ISSUE THREAD

**Issue by mtlynch** [codeapprove/feedback#12]
I'm acting as author on a review. My reviewer has completed their round 1 of their notes, and I'm pushing commits as I respond to different notes. I'd like to see what my reviewer will see on round 2.

If I select "Show Since my last review," CodeApprove shows my changes against the base branch (`master`):

![image](https://user-images.githubusercontent.com/7783288/179776062-f7423f50-6013-40ee-bae9-0de3519c6555.png)

What I want to see are changes since my reviewer's review.

In other words, if the commits are:

* A
* B
* C <- reviewer reviews this version
* D
* E

Then I'd like to see C as the base commit against the latest commit.

**Comment by hatboysam** [comment]
Great feedback. I think the correct solution here would be for that dropdown to let you pick from many more options (since my last review, since the last review by anyone, etc)

**Comment by samatcodeapprove** [comment]
Ok I expanded this dropdown to now let you choose any review (that had a comment or approval change) as the "since" time, so this should be much better now!
